examples/quantize/README.md

```diff
@@ -34,7 +34,7 @@ Run the quantized model:
 
 ```bash
 # start inference on a gguf model
-./llama-cli -m ./models/mymodel/ggml-model-Q4_K_M.gguf -n 128
+./llama-cli -m ./models/mymodel/ggml-model-Q4_K_M.gguf -n 128 -cnv -p "You are a helpful assistant"
```

Comment:
I think `-n 128` can be removed, because in conversation mode we expect the model to stop at EOG/EOS token

Reply:
Done